options/locale/locale_en-US.ini

```diff
@@ -3389,6 +3389,7 @@ management = Secrets Management
 
 [actions]
 actions = Actions
+rerun = Rerun
```

Comment:
Even though it is unlikely to be used elsewhere, I would move it to top-level, e.g. `rerun` instead of `actions.rerun`.

Reply:
Another idea might be to call it `Restart` which I think it a more commonly used word.

Reply:
Done